packages/cmr-client/src/getUrl.ts

```diff
@@ -0,0 +1,56 @@
+function getCmrHost(cmrEnvironment?: string): string {
+  switch (cmrEnvironment) {
+    case 'OPS':
+      return 'cmr.earthdata.nasa.gov';
+    case 'UAT':
+      return 'cmr.uat.earthdata.nasa.gov';
+    case 'SIT':
+      return 'cmr.sit.earthdata.nasa.gov';
+    default:
+      throw new TypeError(`Invalid CMR environment: ${cmrEnvironment}`);
+  }
+}
+
+export function getIngestUrl({
+  host,
+  cmrEnv = process.env.CMR_ENVIRONMENT,
+  provider,
+}: {
+  host?: string,
+  cmrEnv?: string,
+  provider: string,
+}): string {
+  return `https://${host ?? getCmrHost(cmrEnv)}/ingest/providers/${provider}/`;
+}
+
+export function getSearchUrl({
+  host,
+  cmrEnv = process.env.CMR_ENVIRONMENT,
+}: {
+  host?: string,
+  cmrEnv?: string,
+} = {}): string {
+  return `https://${host ?? getCmrHost(cmrEnv)}/search/`;
+}
+
+export function getTokenUrl({
```

Comment:
Clearly you removed getUrl, just want to make sure we're not still relying on it anywhere

Reply:
I _thought_ I had, but I must not have committed that work or something. This commit removes the remaining references to `getUrl`: 8bc02f4f3